docs/csharp/toc.yml

```diff
@@ -998,14 +998,8 @@ items:
       items:
       - name: namespace
         href: language-reference/keywords/namespace.md
-      - name: using
-        items:
-        - name: Contexts for using
-          href: language-reference/keywords/using.md
```

Comment:
While not in TOC, it's still referenced from the [keywords overview page](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/)

As for F1 experience, it looks to have become smarter and brings to the correct page (either statement or directive) depending on the context.